resources/providers/awslib/eks_provider.go

```diff
@@ -20,35 +20,39 @@ package awslib
 import (
 	"context"
 	"fmt"
+	"github.com/aws/aws-sdk-go-v2/service/eks/types"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/eks"
 )
 
+type EksClusterOutput eks.DescribeClusterOutput
+type EksCluster types.Cluster
```

Comment:
I think that we don't use the field anymore, are we?

Reply:
we are using it in the tests